parallel-install/pkg/helm/client.go

```diff
@@ -1,3 +1,7 @@
+//Package helm implements a wrapper over native Helm client.
```

Comment:
```suggestion
//"Package helm" implements a wrapper over a native Helm client.
```

Reply:
done without double-colons, these are not used in go doc in a package doc.